internal/controllers/language_detect/language_detect.go

```diff
@@ -235,3 +239,73 @@ func (ld *LanguageDetect) isBatFileOrShellFile(lang string) bool {
 	return strings.EqualFold(lang, "Batchfile") ||
 		strings.EqualFold(lang, "Shell")
 }
+
+// copyGitFolderWhenIsSubmodule check if the analysis is using with GitHistory enabled,
```

Comment:
Shouldn't be "is running with GitHistory enabled..."?

Reply:
i can change, how you consider correctly text?

Reply:
I think that just change `is using` for `is running`.  "if the analysis is using with GitHistory" sounds a bit confuse to me.

Reply:
done